src/terra-dev-site/doc/application/how-to.2/use-translated-strings.doc.mdx

```diff
@@ -1,25 +1,26 @@
 # How To Use Translated Strings
 
-Translated strings can be retrieved using the `intl` object provided by `injectIntl` HOC-generator provided by [react-intl](https://github.com/formatjs/react-intl/blob/master/docs/API.md#injectintl).
+Translated strings can be retrieved using the [ApplicationIntlContext](/components/terra-application/application/reference/contexts/application-intl-context). An `ApplicationIntlContext.Provider` is configured and provided by [ApplicationBase](/components/terra-application/application/reference/components/application-base). The strings will be translated based upon the `locale` prop provided to ApplicationBase.
+
+In addition to the ApplicationIntlContext, the components provided by react-intl can be used directly. However, the ApplicationIntlContext can be used by Hooks-based components to access i18n capabilities without having to use the `injectIntl` HOC-generator from react-intl.
 
 ```jsx
```

Comment:
I think this example is useful for the how-to, however, looking at this git-diff, I think it'd be helpful to pull this diff forward for consumers that want to upgrade existing content to use this new API and want to know how that should be done. We could render thee example in a collapsed section labeled something like "Upgrade to use ApplicationIntlContext"